src/_includes/get-sdk.md

```diff
@@ -1,9 +1,6 @@
 Once you’re ready to move beyond DartPad and develop real apps,
 you need the Dart SDK.
 
-As you install, **note the path to the SDK.**
-You'll need it in the next step.
```

Comment:
I've removed this sentence because, AFAICT, nowhere where this file is included, is there a "next step" that explicitly asks for the "path to the SDK".